src/Escaper/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+project(Escaper VERSION 1.0.0 LANGUAGES C)
+
+add_library(Escaper STATIC)
+target_sources(Escaper
+  PRIVATE    Escaper.c
+  PUBLIC     Escaper.h)
+
+target_include_directories(Escaper
+  PUBLIC     ${CMAKE_CURRENT_SOURCE_DIR})
+  
+target_link_libraries(Escaper
+  PRIVATE    common_build_options
+  PUBLIC     TASTE::Broker)
```

Comment:
I think that Broker is not required after API change.

Reply:
Yes, I agree. I will delete that.

Nevertheless, I need to add 

target_include_directories(Escaper
  PUBLIC     ${CMAKE_SOURCE_DIR}/Broker)

To be able to include Broker.h in Escaper.h (to get BROKER_BUFFER_SIZE definition)

Reply:
Ok, removed, so it's finished